portfolio/src/main/webapp/script.js

```diff
@@ -52,8 +52,15 @@ function toggleImages() {
   });
 }
 
-async function getMessage() {
+async function getMessages() {
   const response = await fetch('/data');
-  const message = await response.text();
-  document.querySelector('#greeting').innerHTML = message;
+  const quotes = await response.json();
+
+  var messages = '';
```

Comment:
(fyi)
you can use quotes.reduce() here for one liner

Reply:
Changed to reduce